internal/crypto/Cargo.toml

```diff
@@ -98,9 +99,6 @@ version = "0.4.39"
 default-features = false
 features = ["now"]
 
-[target.'cfg(any(not(target_arch = "wasm32"), target_os = "wasi"))'.dependencies]
```

Comment:
Per @cdmurph32's suggestion, this reverts the dependency to what was there prior to #934.

Reply:
… but now uses 0.24.0.